src/preferences/broadcastprofile.cpp

```diff
@@ -294,6 +297,16 @@ bool BroadcastProfile::loadValues(const QString& filename) {
     m_host = XmlParse::selectNodeQString(doc, kHost);
     m_port = XmlParse::selectNodeInt(doc, kPort);
     m_serverType = XmlParse::selectNodeQString(doc, kServertype);
+    switch (static_cast<EncryptionMode>(XmlParse::selectNodeInt(doc, kEncryptionMode))) {
+    case EncryptionMode::Preferred:
+        m_encryptionMode = EncryptionMode::Preferred;
+        break;
+    case EncryptionMode::Disabled:
+        m_encryptionMode = EncryptionMode::Disabled;
+        break;
+    default:
```

Comment:
What is the migration path? I think we should keep "Disabled" after upgrade. We may consider to keep the "not set" state that we have the chance to ask the user in pop up box in a later PR (or in this if you like)

Reply:
I don't mind, my idea was to just check if this new option exists or not.